gemhardware/include/gem/hw/HwGenericAMC.h

```diff
@@ -398,98 +369,103 @@ namespace gem {
          * @param davTO value to use for the DAV timeout
          * @param ttsOverride value to use for the TTS override
          */
-        virtual void resetDAQLink(uint32_t const& davTO=0x500, uint32_t const& ttsOverride=0x0);
+        void resetDAQLink(uint32_t const& davTO=0x500, uint32_t const& ttsOverride=0x0);
 
         /**
          * @returns Returns the 32 bit word corresponding to the DAQ link control register
          */
-        virtual uint32_t getDAQLinkControl();
+        uint32_t getDAQLinkControl();
```

Comment:
`const`?

Reply:
Probably, but will/should be done in a future `const`ification PR, probably during `xdaq15` migration